target/scripts/build/packages.sh

```diff
@@ -201,6 +201,18 @@ function _install_fail2ban
   sedfile -i -r 's/^_nft_add_set = .+/_nft_add_set = <nftables> add set <table_family> <table> <addr_set> \\{ type <addr_type>\\; flags interval\\; \\}/' /etc/fail2ban/action.d/nftables.conf
 }
 
+# Presently the getmail6 package is v6.14, which is too old.
+# v6.18 contains fixes for Google and Microsoft OAuth support.
+# using pip to install getmail.
+function _install_getmail
+{
+  _log 'debug' 'Installing getmail6'
+  apt-get "${QUIET}" --no-install-recommends install python3-pip
+  pip3 install 'getmail6~=6.18.12'
+  ln -s /usr/local/bin/getmail /usr/bin/getmail
+  ln -s /usr/local/bin/getmail-gmail-xoauth-tokens /usr/bin/getmail-gmail-xoauth-tokens
```

Comment:
AFAIK `/usr/bin/` is already in `${PATH}`.

```suggestion
  apt-get "${QUIET}" purge python3-pip
  apt-get "${QUIET}" autoremove
```

Reply:
those are linking from /usr/local/bin into /usr/bin ... you say /usr/bin is in the path but is /usr/local/bin ... Based on my testing I don't believe so.
I'll admit not having tested the requirement to link getmail into /usr/bin but if you don't link getmail-gmail-xauth-tokens then the getmail rc file has to specify the full path in the password_command option.

Reply:
```console
root@mail:/# echo "${PATH}"
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

There really is no difference and we do not need the `ln`, I'm fairly certain.

Reply:
Well the reason I put the line
` ln -s /usr/local/bin/getmail-gmail-xoauth-tokens /usr/bin/getmail-gmail-xoauth-tokens`
was because without it I got errors with this in my config : 
`password_command = ("getmail-gmail-xoauth-tokens", "/var/lib/getmail/configfile.json")`
the error I got was : 
`Configuration error: Program "getmail-gmail-xoauth-tokens" not found`

and the reason the /usr/bin/getmail link is there is because the cron job calls it from there:
`/usr/bin/getmail --getmaildir /var/lib/getmail --rcfile "${FILE}"`

Ultimately it is the difference between the deb install which put the files in /usr/bin/ and the  pip install which put them in /usr/local/bin/

If you really don't want the links then getmail-cron and debug-getmail will need to be updated and we should put a note in the docs that examples given in the links to getmail6's docks wont work as is and people will have to put in the full path to the getmail-gmail-xoauth-tokens helper.

Reply:
I see; then we should change to `/usr/bin/` in our scripts, and re-introduce the links as well though to make us easy. Sorry for the noise then; will you take care of this or shall I do it?

Reply:
I re-applied the `ln` commands in [5b19dbc](https://github.com/docker-mailserver/docker-mailserver/pull/2803/commits/5b19dbc91ff702eb8beced5d5eeb464209d9e7fe)